### PR TITLE
Solution added for 1.2 Limits Numerically (LT2) except the last one.

### PR DESCRIPTION
**Pull request by mlamich1**
I added the solution for  1.2 Limits Numerically (LT2) except the last one.

### CODE REVIEW

File: source/calculus/source/01-LT/02.ptx [hunk 1]
```diff
@@ -341,13 +377,26 @@ the function as <m>x</m> tends to 2?
...
+          </p>
+         </answer>
+         </task>   
  <task> <statement>
         <p>Would your conclusion that resulted from <xref ref = "activity-limits-numerically5"/> change if the function was <m>f(x) = \cos(1/x)</m> or <m>f(x) = \tan(1/x)</m>?</p>
-        </statement> </task>   
+        </statement>
+        <answer>
+          <p>
+            Yes, the limit of  <m>f(x) = \tan(1/x)</m> would be 0 as <m> x </m> tends to 0 however <m>f(x) = \cos(1/x)</m> oscillates as <m> x </m> tends to 0 .
```
Comment (jbunn3) [inline source/calculus/source/01-LT/02.ptx:392]: Both limits still don't exists. I think you were thinking of x tending to infinity for the tan one.

File: source/calculus/source/01-LT/02.ptx [hunk 2]
```diff
@@ -466,15 +517,45 @@ The limit is <m>-0.7</m>.
         </table>
 </introduction>
      <task> <statement> <p>What was the average velocity on the first 50 meters? On the second 50 meters? </p>
-         </statement> </task>  
+         </statement> 
+         <answer>
+          <p>
+            The average velocity on the first 50 meters is <m>10.95 </m> meter per second  and   <m>11.93 </m> meter per second on the second 50 meters.
```
Comment (jbunn3) [inline source/calculus/source/01-LT/02.ptx:523]:
```suggestion
            The average velocity on the first 50 meters is <m>9.09</m> meters per second  and   <m>11.93 </m> meters per second on the second 50 meters.
```

File: source/calculus/source/01-LT/02.ptx [hunk 3]
```diff
@@ -466,15 +517,45 @@ The limit is <m>-0.7</m>.
...
+          </p>
+         </answer>
+         </task>  
                  <task> <statement> <p>What was the average velocity between 30 and 50 meters? Between 50 and 70 meters? </p>
-                  </statement> </task>    
+                  </statement>
+                  <answer>
+          <p>
+            The average velocity between 30 and 50 meters is   <m>11.62 </m> meter per second and   <m>12.19 </m> meter per second between 50 and 70 meters.
```
Comment (jbunn3) [inline source/calculus/source/01-LT/02.ptx:531]:
```suggestion
            The average velocity between 30 and 50 meters is   <m>11.62 </m> meters per second and   <m>12.19 </m> meters per second between 50 and 70 meters.
```

File: source/calculus/source/01-LT/02.ptx [hunk 4]
```diff
@@ -466,15 +517,45 @@ The limit is <m>-0.7</m>.
...
+          </p>
+         </answer>
+                   </task>    
                  <task> <statement> <p> What was the average velocity between 40 and 50 meters? Between 50 and 60 meters? </p>
-         </statement> </task> 
+         </statement>
+          <answer>
+          <p>
+            The average velocity between 40 and 50 meters is   <m>11.76 </m> meter per second and   <m>12.19 </m> meter per second between 50 and 70 meters.
```
Comment (jbunn3) [inline source/calculus/source/01-LT/02.ptx:539]:
```suggestion
            The average velocity between 40 and 50 meters is   <m>11.76 </m> meters per second and   <m>12.20 </m> meters per second between 50 and 60 meters.
```

**Comment by mlamich1** [comment]
Thanks Jared.

File: source/calculus/source/01-LT/02.ptx [hunk 5]
```diff
@@ -466,15 +517,45 @@ The limit is <m>-0.7</m>.
...
+            <m>12</m> meter per second.
+          </p>
+         </answer>
+          </task> 
                <task> <statement> <p>Using the table of values, explain why 50 meters is NOT the best guess for when the instantaneous velocity was the largest. What other point would be more reasonable? </p>
-         </statement> </task> 
+         </statement>
+        <answer>
+          <p>
+            <m>12</m>
```
Comment (jbunn3) [inline source/calculus/source/01-LT/02.ptx:555]: I think the intention is to have students analyze the average velocities on all 10 subintervals to see that he's still speeding up at 50 meters. Here's the data from excel: 
table

File: source/calculus/source/01-LT/02.ptx (hunk 1 above)
Comment (jbunn3) [inline source/calculus/source/01-LT/02.ptx:392]:
```suggestion
            No, both <m>f(x) = \tan(1/x)</m> and <m>f(x) = \cos(1/x)</m> oscillate as <m> x </m> tends to 0. 
```

File: source/calculus/source/01-LT/02.ptx (hunk 5 above)
Comment (jbunn3) [inline source/calculus/source/01-LT/02.ptx:555]:
```suggestion
            <m>60</m>, <m>70</m>, or <m>80</m> meters are all more reasonable.
```

**Review by jbunn3 — APPROVED** [review]
All looks good to me now!

**Pull request merged**